docs/reference/configuration/raftstore-config.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Raftstore Configuration 
+summary: Learn about Raftstore configuration in TiKV.
+category: reference
+---
+
+# Raftstore Configurations
+
+Raftstore is TiKV's implementation of [Multi-raft](https://tikv.org/deep-dive/scalability/multi-raft/) to manage multiple Raft consensus groups on one node. Raftstore is comprised of two major components:
```

Comment:
I don't think it's correct. Multi-Raft is to manage a property (key space in TiKV's case) by multiple raft groups instead of single one. There is no such a universal node that manage multiple groups at once.

Reply:
Hmmm yes this could be clearer, maybe "...to allow multiple Raft consensus groups to be resident in one node"

Reply:
@BusyJay  Can you write several sentences describing Raftstore?

Reply:
...to manage multiple raft peers on one node.